subxt/src/lib.rs

```diff
@@ -168,46 +168,29 @@ pub mod ext {
 ///
 /// ## `substitute_type(path = "...", with = "...")`
 ///
-/// This attribute replaces any reference to the generated type at the path given by `path` with a
-/// reference to the path given by `with`.
+/// Replaces references to a specific type in the generated code with a custom type.
+///
+/// ### Important Notes
+/// - The path to replace must be an absolute path (fully qualified)
+/// - The replacement type must implement the necessary traits for encoding and decoding
```

Comment:
nit: Maybe we can mention the traits here?

Reply:
Woudln't that be dynamic?

Reply:
@lexnv